en/orm/associations.rst

```diff
@@ -357,6 +357,23 @@ syntax::
         }
     }
 
+Sometimes you may want to configure composite keys in your associations::
+
+    $this->hasMany('Reviews', [
+        'foreignKey' => [
+            'article_id',
+            'article_name'
+        ],
+        'bindingKey' => [
+            'article_id',
+            'article_name'
+        ]
+    ]);
+
```

Comment:
Would this make it easier to understand the concept?
```php
// within ArticlesTable::initialize() call
$this->hasMany('Reviews', [
    'foreignKey' => [
        'article_id',
        'article_revision'
    ],
    'bindingKey' => [
        'id',
        'revision'
    ]
]);
```
Correct me if I missunderstood binding keys.

p.s.: I think usually bindingKeys are not required to be set as they are by default set to whatever the primaryKey is set to (which can also be a composite key).